src/main/java/io/github/moulberry/notenoughupdates/core/GuiElementTextField.java

```diff
@@ -395,6 +396,55 @@ public void keyTyped(char typedChar, int keyCode) {
 							textField.setCursorPosition(pos + 1);
 						}
 					}
+				} else {
+					for (int i = 0; i < 10; i++) {
+						if (typedChar == Integer.toString(i + 1).charAt(0)) {
+							int pos = textField.getCursorPosition() - 2;
+							if (pos >= 0 && pos < textField.getText().length()) {
+								if (textField.getText().charAt(pos) == '*') {
+									switch (i) {
+										case 0:
+											masterStarUnicode = "\u278A";
+											break;
+										case 1:
+											masterStarUnicode = "\u278B";
+											break;
+										case 2:
+											masterStarUnicode = "\u278C";
+											break;
+										case 3:
+											masterStarUnicode = "\u278D";
+											break;
+										case 4:
+											masterStarUnicode = "\u278E";
+											break;
+										case 5:
+											masterStarUnicode = "\u278F";
+											break;
+										case 6:
+											masterStarUnicode = "\u2790";
+											break;
+										case 7:
+											masterStarUnicode = "\u2791";
+											break;
+										case 8:
+											masterStarUnicode = "\u2792";
+											break;
+										case 9:
+											masterStarUnicode = "\u2793";
+											break;
+									}
```

Comment:
I wish enhanced switch statements were in java 8

Reply:
uh?
 Unresolved conversations
1 conversation must be resolved before merging.

Reply:
didn't ask